### ISSUE TITLE
Take screenshots?

### ISSUE THREAD

**Issue by oband2** [kolton/d2bot-with-kolbot#1356]
Wasnt able to find anything about this so here we go.

I know the bot takes screenshot if its hostiled and kills someone, but is there any way i can program it so it will also take screenshot whenever i pick up a HR? would like to see where i find it exactly, area is not enough.

can i do this? what would the code look like, and where to place it?
ty in advance.

**Comment by noah-** [comment]
takescreenshot();

you would need to add this in the pickit code somewhere with some conditions when it should take the screenshot

**Comment by oband2** [comment]
so like this?
takescreenshot(); [name] >= pulrune && [name] <= zodrune
?

**Comment by noah-** [comment]
no, it would need to be in the code that checks when items are kept
you can't add it to the .nip files themselves, though I like that as an feature idea, adding predicates to pickits

**Comment by oband2** [comment]
So what you're saying is, that its unable to function atm?

**Comment by noah-** [comment]
takescreenshot functions, you need to modify kolbot pick items code (.js file) to get the feature you want, you can't add it as a condition in the nip file

**Comment by oband2** [comment]
Any chance u can be more specific about how to do it when picking up hrs? what would the code look like, and where to place it? sorry but im pretty retarded about creating these codes and how it exactly works

**Comment by mf022** [comment]
maybe you should look to Pickit.js, ~lines 158 ++
```
					// Item can fit - pick it up
					if (canFit) {
						this.pickItem(pickList[0], status.result, status.line);
						takescreenshot();
					}
```
